Pod/Classes/SwiftPhotoGallery.swift

```diff
@@ -143,11 +143,20 @@ public class SwiftPhotoGallery: UIViewController, UICollectionViewDataSource, UI
 
 
     // MARK: UICollectionViewDelegate Methods
+
+    public func scrollViewWillBeginDragging(scrollView: UIScrollView) {
+        pageControl.alpha = 1.0
+    }
     
     public func scrollViewDidEndDecelerating(scrollView: UIScrollView) {
 
         // If the scroll animation ended, update the page control to reflect the current page we are on
         pageControl.currentPage = currentPage
+
+        // Auto-hide the pageControl
+        UIScrollView.animateWithDuration(1.0, delay: 2.0, options: nil, animations: { () -> Void in
```

Comment:
The `animateWithDuration` class method is really on `UIView`, not `UIScrollView`. This works because `UIScrollView` is a subclass of `UIView`, but normally you would just use `UIView` for clarity. 

https://developer.apple.com/library/ios/documentation/WindowsViews/Conceptual/ViewPG_iPhoneOS/AnimatingViews/AnimatingViews.html#//apple_ref/doc/uid/TP40009503-CH6-SW1